pages/pipelines/command_step.md.erb

```diff
@@ -120,12 +120,51 @@ _Optional attributes:_
       <em>Example:</em> <code>"My reason"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>retry</code></td>
+    <td>
+      Configuration of how retries (either <strong>automatic</strong> or <strong>manual</strong>) operate on this step<br>
+      Note: If automatic retries are enabled, a retry job will be triggered if it fails with any exit status <code>"*"</code> a maximum of 2 times.<br>
+
+      <br/>
+      <em>Example (automatic retries with defaults):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic: true</code><br/>
+
+      <br/>
+      <em>Example (different automatic retry rules depending on exit status):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic:</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: 5</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;limit: 2</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: "*"</code><br/>
```

Comment:
Does `exit_status` always need to have a `-`in front of it?

Reply:
Yeah, in this case, we're defining a list of "automatic retry rules". There are 2 rules defined:

```
- exit_status: 5
   limit: 2
- exit_status: "*"
   limit: ...
```

The `-` means its a YAML array. Just like:

```
steps:
  - command: "..."
     foo: "bar"
```

Reply:
Okay cool. So how about `limit`? It looks like that one can be standalone and doesn't need a `-`?

Reply:
`automatic` can accept either a single retry rule (no `-`) or multiple retry rules (which need a `-`).

So:

```
# Changes the default automatic retry rule to have a limit of 4.
automatic:
  limit: 4

# Changes the default automatic retry rule to only look at jobs with an exit status of 23.
automatic:
  exit_status: 23

# When defining an array of exit status rules, both `exit_status` and `limit` are required. So this will raise an error...
automatic:
  - exit_stauts: 23

# As will this
automatic:
  - limit: 2
```

Reply:
Oh, and a single retry rule looks like this:

```
automatic:
  limit: 4
  exit_status: 2
```

If they want to add another rule, they'll need to convert it to a list:

```
automatic:
  - limit: 4
    exit_status: 2
  - limit: 2
    exit_status: 5
```